lab-1/Question-3.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html >
+<head>
+    <title>Question-3</title>
+</head>
+<body>
+    
+        <h1 style="padding-left: 22%;">World's best scrambles Eggs</h1>
+        <p>This recipe is a very special breakfast: just look at he ingredients to see why it has to be tasted to be believe</p>
+        <h2>Ingredients</h2>
+        <p>The following ingredients make one serving</p>
+
+        <ul>
+            <li type="disc">2 Eggs</li>
```

Comment:
you can define the "type" within the ul tag

Reply:
ok